mypyc/test-data/run-attrs-non-auto.test

```diff
@@ -0,0 +1,142 @@
+-- Test cases for dataclasses based on the attrs library
```

Comment:
Maybe explicitly mention it here that these test cases don't use `auto_attribs=True` for extra clarity?

Reply:
done